v1.20/inspur-iop-arm64/id_rsa-arm

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Might want to verify where you use this private key.
Probably don't want to upload this.